### PR TITLE
Update deprecated circleci images to latest

### PR DESCRIPTION
**Pull request by yorinasub17**
Update deprecated circleci images to latest

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -112,8 +112,8 @@ jobs:
             - $HOME/go/src/
 
       # The weird way you have to set PATH in Circle 2.0
-      - run: echo 'export PATH=$HOME/.local/bin:$HOME/go/bin:$HOME/terraform:$HOME/packer:$PATH'
-          >> $BASH_ENV
+      - run: |
```
Comment (marinalimeira) [inline .circleci/config.yml:115]: What difference does it make to use the `|`?

**Review by marinalimeira — APPROVED** [review]
LGTM

File: .circleci/config.yml (hunk 1 above)
Comment (yorinasub17) [reply .circleci/config.yml:115]: `|` ensures the string is treated as "raw" multiline, and prevents scripts from reflowing the text to keep column width.

**Comment by yorinasub17** [comment]
Thanks for review! Merging now.

**Pull request merged**